### PR TITLE
Store and load JUnit tests

### PR DESCRIPTION
**Pull request by OzairHasan**

### CODE REVIEW

File: src/test/java/seedu/allonus/storage/ContactsManagerStub.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import seedu.allonus.contacts.entry.Faculty;
+import seedu.allonus.contacts.entry.Name;
+import seedu.allonus.contacts.entry.Description;
+
+import java.util.ArrayList;
+
+/**
+ * Stub class for ContactsManager for unit testing.
+ */
+public class ContactsManagerStub extends ContactsManager {
```
Comment (wli-linda) [inline src/test/java/seedu/allonus/storage/ContactsManagerStub.java:15]: Nice job on creating a stub for unit testing!

**Review by wli-linda — COMMENTED** [review]
LGTM

File: src/test/java/seedu/allonus/storage/StorageFileTest.java [hunk 2]
```diff
@@ -1,11 +1,166 @@
...
+        for (int i = 3; i < studyManager.getModulesList().size(); i++) {
+            assertEquals(storageFile.getModuleInFileFormat(i), "S," + modules[i - 3]);
+        }
+        for (int i = 3; i < expenseTracker.getExpenseList().size(); i++) {
+            assertEquals(storageFile.getExpenseInFileFormat(i), "E," + expenses[i - 3]);
+        }
+        for (int i = 3; i < contactsManager.getContactsList().size(); i++) {
+            assertEquals(storageFile.getContactInFileFormat(i), "C," + contacts[i - 3]);
+        }
+    }
```
Comment (YitHien) [inline src/test/java/seedu/allonus/storage/StorageFileTest.java:165]: Good use of stubs to test load and store methods!

**Review by YitHien — COMMENTED** [review]
LGTM!

File: src/test/java/seedu/allonus/storage/ExpenseTrackerStub.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import seedu.allonus.expense.Expense;
+import seedu.allonus.expense.ExpenseTracker;
+
+import java.util.ArrayList;
+
+/**
+ * Stub class for ExpenseTracker for unit testing.
+ */
+public class ExpenseTrackerStub extends ExpenseTracker {
```
Comment (mathanmahe) [inline src/test/java/seedu/allonus/storage/ExpenseTrackerStub.java:11]: Nice use of stubs to work around the JUnit testing error!

**Review by mathanmahe — APPROVED** [review]
Looks good to merge!

**Pull request merged**